pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -734,23 +735,14 @@ private void setupPubSub(@NonNull Config cfg, NotificationUtil notificationUtil)
 
         logger.info("Setting up pubsub for {}/{}", projectId, dsmToDssSubscriptionId);
 
-        try {
-            PubSubResultMessageSubscription.dssToDsmSubscriber(projectId, dsmToDssSubscriptionId);
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
+        Try.evaluate(() -> PubSubResultMessageSubscription.dssToDsmSubscriber(projectId, dsmToDssSubscriptionId))
+                .ifThrowsThenRunTask(Exception::printStackTrace, Exception.class);
 
-        try {
-            DSMtasksSubscription.subscribeDSMtasks(projectId, dsmTasksSubscriptionId);
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
+        Try.evaluate(() -> DSMtasksSubscription.subscribeDSMtasks(projectId, dsmTasksSubscriptionId))
+                .ifThrowsThenRunTask(Exception::printStackTrace, Exception.class);
```

Comment:
Why not use Cyclops Try for those needs? It implements Try monade and covers the functionality.

Reply:
@Trimax Yep, I figured that it's more of a fun side project for educational purposes which also can be useful, but since there are other open source solutions I decided to close this. Thank you!